src/test/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseControllerIntTest.java

```diff
@@ -505,14 +528,13 @@ private void validateResponse(ValidatableResponse validatableResponse, String st
             .body("offences[0].offenceSummary", equalTo("On 01/01/2015 at own, stole article, to the value of £987.00, belonging to person."))
             .body("offences[0].act", equalTo("Contrary to section 1(1) and 7 of the Theft Act 1968."))
             .body("offences[1].offenceTitle", equalTo("Theft from a different shop"))
-            .body("probationStatus", equalTo(PROBATION_STATUS))
-            .body("probationStatusActual", equalTo(null))
-            .body("previouslyKnownTerminationDate", equalTo(LocalDate.of(2010, 1, 1).format(DateTimeFormatter.ISO_LOCAL_DATE)))
-            .body("suspendedSentenceOrder", equalTo(true))
-            .body("breach", equalTo(true))
+            .body("probationStatus", equalTo("Possible NDelius record"))
+//            .body("probationStatusActual", equalTo("CURRENT"))
```

Comment:
Think this probably wasn't meant to be commented out - will uncomment and fix in a follow up commit